.github/actions/setup-spack-mirror/action.yaml

```diff
@@ -0,0 +1,38 @@
+name: 'setup-spack-mirror'
+description: 'Creates a spack mirror.'
+
+runs:
+  using: "composite"
+
+  steps:
+
+  - name: os-setup
+    uses: ./.github/actions/setup-os
+
+  - name: cache-spack-mirror
+    id: cache-spack-mirror
+    uses: actions/cache@v2
+    with:
+      path: ~/spack-mirror
+      key: spack-mirror.${{ runner.os }}3
+
+  - name: create-spack-mirror
+    shell: bash
+    if: steps.cache-spack-mirror.outputs.cache-hit != 'true'
+    run: |
+      source ./setup.sh
+      # DH* TODO - REPLACE jedi-ufs-all with a better environment
```

Comment:
Not this time - a follow-up when we clean up our various env packages and templates.

Reply:
wasn't sure about this "jedi-ufs-all" but based on your comment here, it seems like a change will come to this in a future PR post-release.